server/templates/task-activity-new.html

```diff
@@ -33,10 +33,10 @@
       type="text"
       name="Location"
       {{ if .plans }}
-      {{ $plan := index .plans $.selectedPlan }}
-      {{ if ne "" $plan.Location }}value="{{ $plan.Location }}"{{ end }}
+        {{ $plan := index .plans $.selectedPlan }}
+        {{ if ne "" $plan.Location }}value="{{ $plan.Location }}"{{ end }}
       {{ else }}
-      {{ $.latestActivity.Location }}
+        value="{{ $.latestActivity.Location }}"
```

Comment:
Good catch. Please remove the indent though - the style here is to indent for HTML/CSS only, not for template engine things.

Reply:
https://github.com/nyiyui/jks/pull/2/commits/d66be7b68dc6fc1bd3a34c0dcbb58c3734983a54